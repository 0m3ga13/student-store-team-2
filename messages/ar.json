{
    "Index": {
        "Title": "المتجر الصغير",
        "Home": "الرئيسية",
        "Products": "المنتجات",
        "Orders": "الطلبيات",
        "State": "الحالة",
        "Buy": "شراء",
        "Borrow": "إعارة",
        "Description": "هذه المنصة هي مربحة للجميع: القرضاء يحققون دخلاً إضافيًا، والمقترضون يقللون التكاليف.",
        "Profile": "الحسـاب",
        "For Sale": "للبيع",
        "For Borrow": "للإعارة",
        "Sale & Borrow": "إعارة وشراء",
        "Day": "لليوم",
        "How It Works": "طريـقة العمل",
        "Hello": "مرحبا",
        "Sign out": "تسجيل الخروج",
        "Sign In": "تسجيل الدخول",
        "Login with Google": "سحل الدخول بحسابك",
        "Sign Up With Google": "سحل الدخول بحسابك"
    }
}
