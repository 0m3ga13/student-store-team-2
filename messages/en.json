{
    "Index": {
        "Title": "MiniStore",
        "Home": "Home",
        "Products": "Products",
        "Orders": "Orders",
        "State": "State",
        "Donate": "Donate",
        "Buy": "Buy",
        "Borrow": "Borrow",
        "Description": "This platform is a win-win for everyone involved lenders earn extra income, borrowers cut costs.",
        "Profile": "Profile",
        "for_sale": "for_sale",
        "for_borrow": "for_borrow",
        "Sale & Borrow": "Sale & Borrow",
        "Day": "Day",
        "How It Works": "How It Works",
        "recently added": "recently added",
        "Hello": "Hello",
        "Sign out": "Sign out",
        "Sign In": "Sign In",
        "Login with Google": "Login with Google",
        "Sign Up With Google": "Sign Up With Google",
        "Free delivery": "Free delivery",
        "feature descriprion 1": "You can receive your order without incurring any additional shipping costs",
        "qualite guarantee": "qualite guarantee",
        "feature descriprion 2": "Our products or services meet high standards and perform as promised.",
        "daily offers": "daily offers",
        "feature descriprion 3": "Exclusive discounts and promotions on a daily basis",
        "100% secure payement": "100% secure payement",
        "feature descriprion 4": "Your financial information is protected and your transactions are safe from unauthorized access",
        "My Dashboard": "My Dashboard",
        "Awaiting Pickup": "Awaiting Pickup",
        "All": "All",
        "Active": "Active",
        "Completed": "Completed",
        "Sort": "Sort:",
        "Newest": "Newest",
        "Oldest": "Oldest",
        "my stuff":"my stuff",
        "borrowings": "borrowings",
        "lendings": "lendings",
        "purchases": "purchases",
        "sales": "sales",
        "My Borrowings": "My Borrowings",
        "My Lendings": "My Lendings",
        "My Purchases": "My Purchases",
        "My Sales": "My Sales",
        "General details": "General details",
        "First Name": "First Name",
        "Last Name": "Last Name",
        "Birth Date": "Birth Date",
        "Gender": "Gender",
        "Institution": "Institution",
        "Contact & address Detail": "Contact & address Detail",
        "Phone Number": "Phone Number",
        "Building": "Building",
        "Street": "Street",
        "City": "City",
        "State_place": "State",
        "Save Changes": "Save Changes",
        "Upload Image": "Upload Image",
        "Close": "Close",
        "Upload": "Upload",
        "Loading": "Loading",
        "Not a member ?": "Not a member ?",
        "Sign Up": "Sign Up",
        "Store": "Store",
        "Create Your": "Create Your",
        "and start exchanging with other students": "and start exchanging with other students",
        "Or Login Using:": "Or Login Using:",
        "Login": "Login",
        "LOGIN": "LOGIN ",
        "Already a member ?": "Already a member ?",
        "Email and password are required": "Email and password are required",
        "Password should be at least 8 characters long": "Password should be at least 8 characters long",
        "and Conditions and Privacy Policy of The website": "and Conditions and Privacy Policy of The website",
        "Email": "Email",
        "Password": "Password",
        "Full Name": "Full Name",
        "Address": "Address",
        "and Privacy Policy": "and Privacy Policy",
        "I have read and agree with all of the Terms": "I have read and agree with all of the Terms",
        "PROFILE": "PROFILE",
        "SECURITY": "SECURITY",
        "Upload ID": "Upload ID",
        "Edit ID": "Edit ID",
        "Edit Image": "Edit Image",
        "WHY DONATE": "WHY DONATE",
        "MAKE A DIFFERENCE BY DONATING": "MAKE A DIFFERENCE BY DONATING",
        "Donate paragraph": "In our fast-paced lives, it's easy to forget the power of gratitude. Students, often on tight budgets, rely on our support. Your donation is more than just financial aid; it's a symbol of gratitude. By giving generously, you're helping students create a brighter future, one contribution at a time.",
        "Small Help": "Small Help",
        "Some Help": "Some Help",
        "Big Help": "Big Help",
        "List an Item": "List an Item",
        "Drag and drop some files here or click to select files": "Drag and drop some files here or click to select files",
        "Location": "Location",
        "Category": "Category",
        "books": "books",
        "kitchen": "kitchen",
        "sports": "sports",
        "games": "games",
        "clothes": "clothes",
        "electronics": "electronics",
        "furniture": "furniture",
        "description": "Description",
        "Product Condition": "Product Condition",
        "Sell": "Sell",
        "Price Per Week": "Price Per Week",
        "Price": "Price",
        "Upload Image(s)": "Upload Image(s)",
        "List": "List",
        "Product Name": "Product Name",
        "Comments": "Comments",
        "Type your comment": "Type your comment",
        "Type your reply": "Type your reply",
        "Respond": "Respond",
        "Like": "Like",
        "Send": "Send",
        "Authentication error": "Invalid login credentials. Please check your email and password.",
        "Filter": "Filter",
        "Pending": "Pending",
        "Quick Links": "Quick Links",
        "About LACITÉ": "ABOUT LACITÉ",
        "About Us": "ABOUT US",
        "Github Project Link": "GITHUB PROJECT LINK",
        "Contact Us": "Contact Us",
        "We Couldn't Find the page you're looking for": "We Couldn't Find the page you're looking for",
        "Go To Home": "Go To Home",
        "We Couldn't Find the requested Product": "We Couldn't Find the requested Product."
    }
}
