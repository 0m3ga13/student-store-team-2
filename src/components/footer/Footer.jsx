"use client";
import Link from "next/link";
import { useTranslations } from "next-intl";

function Footer() {
    const t = useTranslations("Index");
    const contributors = [
        {
            name: "BARKA Oussama",
            link: "https://github.com/0m3ga13",
        },
        {
            name: "Imane BELAID",
            link: "https://github.com/Emybel",
        },
        {
            name: "Mohammed Bennaceur",
            link: "https://github.com/medshk",
        },
        {
            name: "Ahmed Djebnoune",
            link: "https://github.com/Bolphunga",
        },
        {
            name: "Samira TOUBAL SEGHIR",
            link: "https://github.com/samiraTbl",
        },
    ];

    return (
        <footer className='text-sm w-full mx-auto'>
            <div className='flex md:flex-col flex-row md:justify-between justify-center'>
                <div className='flex justify-around items-start flex-col md:flex-row md:flex-wrap w-full text-left'>
                    <div className='flex justify-start flex-col ml-4'>
                        <Link href='/'>
                            <button className='font-bold text-3xl sm:text-base tracking-wide md:hover:tracking-widest transform-all ease-in-out duration-500 font-mono text-black '>
                                {t("Title")}
                            </button>
                        </Link>
                        <p className='mt-10 text-content sm:text-base'>
                            {t("Description")}
                        </p>
                    </div>

                    {/* Created by */}

                    <div className=' flex justify-start flex-col'>
                        <p className=' tracking-wider py-1 my-1'>CONTACT US</p>

                        <div>
                            {contributors.map((contributor, index) => (
                                <Link
                                    key={index}
                                    href={contributor.link}
                                    className='flex flex-row items-center hover:text-accent dark:hover:text-white'
                                >
                                    <svg
                                        className='w-4 h-4 mx-2'
                                        aria-hidden='true'
                                        xmlns='http://www.w3.org/2000/svg'
                                        fill='currentColor'
                                        viewBox='0 0 20 20'
                                    >
                                        <path
                                            fillRule='evenodd'
                                            d='M10 .333A9.911 9.911 0 0 0 6.866 19.65c.5.092.678-.215.678-.477 0-.237-.01-1.017-.014-1.845-2.757.6-3.338-1.169-3.338-1.169a2.627 2.627 0 0 0-1.1-1.451c-.9-.615.07-.6.07-.6a2.084 2.084 0 0 1 1.518 1.021 2.11 2.11 0 0 0 2.884.823c.044-.503.268-.973.63-1.325-2.2-.25-4.516-1.1-4.516-4.9A3.832 3.832 0 0 1 4.7 7.068a3.56 3.56 0 0 1 .095-2.623s.832-.266 2.726 1.016a9.409 9.409 0 0 1 4.962 0c1.89-1.282 2.717-1.016 2.717-1.016.366.83.402 1.768.1 2.623a3.827 3.827 0 0 1 1.02 2.659c0 3.807-2.319 4.644-4.525 4.889a2.366 2.366 0 0 1 .673 1.834c0 1.326-.012 2.394-.012 2.72 0 .263.18.572.681.475A9.911 9.911 0 0 0 10 .333Z'
                                            clipRule='evenodd'
                                        />
                                    </svg>
                                    <p className='py-1'>{contributor.name}</p>
                                </Link>
                            ))}
                        </div>
                    </div>

                    {/* Our Products */}

                    <div className='flex justify-start flex-col uppercase tracking-wide '>
                        <p className='font-semibold tracking-wider py-1'>
                            Quick links
                        </p>

                        <Link
                            href='#'
                            className='py-2 mt-3  no-underline hover:text-accent'
                        >
                            <p>Home</p>
                        </Link>
                        <Link
                            href='#'
                            className='py-2  no-underline hover:text-accent'
                        >
                            <p>Furniture</p>
                        </Link>
                        <Link
                            href='#'
                            className='py-2  no-underline hover:text-accent'
                        >
                            <p>Clothes</p>
                        </Link>
                        <Link
                            href='#'
                            className='py-2  no-underline hover:text-accent'
                        >
                            <p>Electronics</p>
                        </Link>
                        <Link
                            href='#'
                            className='py-2  no-underline hover:text-accent'
                        >
                            <p>gaming</p>
                        </Link>
                    </div>

                    <div className=' flex justify-start flex-col uppercase'>
                        <p className='font-semibold tracking-wide py-1'>
                            ABOUT STUDENT STORE
                        </p>
                        <Link
                            href='#'
                            className='py-2 no-underline hover:text-accent'
                        >
                            <p>ABOUT US</p>
                        </Link>
                        <Link
                            href='#'
                            className='py-2 no-underline hover:text-accent'
                        >
                            <p>HOW IT WORKS!</p>
                        </Link>
                        <Link
                            href='#'
                            className='py-2 no-underline hover:text-accent'
                        >
                            <p>Terms of Use</p>
                        </Link>
                        <Link
                            href='#'
                            className='py-2  no-underline hover:text-accent'
                        >
                            <p>Privacy Policy</p>
                        </Link>
                    </div>
                </div>
            </div>
        </footer>
    );
}

export default Footer;
