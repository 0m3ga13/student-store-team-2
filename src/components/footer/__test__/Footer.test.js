import renderer from "react-test-renderer";

import Footer from "../Footer";
import { NextIntlClientProvider } from "next-intl";
import messages from "../../../../messages/en.json";

it("renders correctly", () => {
    const tree = renderer
        .create(
            <NextIntlClientProvider locale='en' messages={messages}>
                <Footer />
            </NextIntlClientProvider>
        )
        .toJSON();
    expect(tree).toMatchSnapshot();
});
