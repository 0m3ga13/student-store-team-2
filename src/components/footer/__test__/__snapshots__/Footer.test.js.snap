// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<footer
  className="text-sm w-full mx-auto"
>
  <div
    className="flex md:flex-col flex-row md:justify-between justify-center"
  >
    <div
      className="flex justify-around items-start flex-col md:flex-row md:flex-wrap w-full text-left"
    >
      <div
        className="flex justify-start flex-col ml-4"
      >
        <a
          href="/"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onTouchStart={[Function]}
        >
          <button
            className="font-bold text-3xl sm:text-base tracking-wide md:hover:tracking-widest transform-all ease-in-out duration-500 font-mono text-black "
          >
            MiniStore
          </button>
        </a>
        <p
          className="mt-10 text-content sm:text-base"
        >
          This platform is a win-win for everyone involved lenders earn extra income, borrowers cut costs.
        </p>
      </div>
      <div
        className=" flex justify-start flex-col"
      >
        <p
          className=" tracking-wider py-1 my-1"
        >
          CONTACT US
        </p>
        <div>
          <a
            className="flex flex-row items-center hover:text-accent dark:hover:text-white"
            href="https://github.com/0m3ga13"
            onClick={[Function]}
            onMouseEnter={[Function]}
            onTouchStart={[Function]}
          >
            <svg
              aria-hidden="true"
              className="w-4 h-4 mx-2"
              fill="currentColor"
              viewBox="0 0 20 20"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                clipRule="evenodd"
                d="M10 .333A9.911 9.911 0 0 0 6.866 19.65c.5.092.678-.215.678-.477 0-.237-.01-1.017-.014-1.845-2.757.6-3.338-1.169-3.338-1.169a2.627 2.627 0 0 0-1.1-1.451c-.9-.615.07-.6.07-.6a2.084 2.084 0 0 1 1.518 1.021 2.11 2.11 0 0 0 2.884.823c.044-.503.268-.973.63-1.325-2.2-.25-4.516-1.1-4.516-4.9A3.832 3.832 0 0 1 4.7 7.068a3.56 3.56 0 0 1 .095-2.623s.832-.266 2.726 1.016a9.409 9.409 0 0 1 4.962 0c1.89-1.282 2.717-1.016 2.717-1.016.366.83.402 1.768.1 2.623a3.827 3.827 0 0 1 1.02 2.659c0 3.807-2.319 4.644-4.525 4.889a2.366 2.366 0 0 1 .673 1.834c0 1.326-.012 2.394-.012 2.72 0 .263.18.572.681.475A9.911 9.911 0 0 0 10 .333Z"
                fillRule="evenodd"
              />
            </svg>
            <p
              className="py-1"
            >
              BARKA Oussama
            </p>
          </a>
          <a
            className="flex flex-row items-center hover:text-accent dark:hover:text-white"
            href="https://github.com/Emybel"
            onClick={[Function]}
            onMouseEnter={[Function]}
            onTouchStart={[Function]}
          >
            <svg
              aria-hidden="true"
              className="w-4 h-4 mx-2"
              fill="currentColor"
              viewBox="0 0 20 20"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                clipRule="evenodd"
                d="M10 .333A9.911 9.911 0 0 0 6.866 19.65c.5.092.678-.215.678-.477 0-.237-.01-1.017-.014-1.845-2.757.6-3.338-1.169-3.338-1.169a2.627 2.627 0 0 0-1.1-1.451c-.9-.615.07-.6.07-.6a2.084 2.084 0 0 1 1.518 1.021 2.11 2.11 0 0 0 2.884.823c.044-.503.268-.973.63-1.325-2.2-.25-4.516-1.1-4.516-4.9A3.832 3.832 0 0 1 4.7 7.068a3.56 3.56 0 0 1 .095-2.623s.832-.266 2.726 1.016a9.409 9.409 0 0 1 4.962 0c1.89-1.282 2.717-1.016 2.717-1.016.366.83.402 1.768.1 2.623a3.827 3.827 0 0 1 1.02 2.659c0 3.807-2.319 4.644-4.525 4.889a2.366 2.366 0 0 1 .673 1.834c0 1.326-.012 2.394-.012 2.72 0 .263.18.572.681.475A9.911 9.911 0 0 0 10 .333Z"
                fillRule="evenodd"
              />
            </svg>
            <p
              className="py-1"
            >
              Imane BELAID
            </p>
          </a>
          <a
            className="flex flex-row items-center hover:text-accent dark:hover:text-white"
            href="https://github.com/medshk"
            onClick={[Function]}
            onMouseEnter={[Function]}
            onTouchStart={[Function]}
          >
            <svg
              aria-hidden="true"
              className="w-4 h-4 mx-2"
              fill="currentColor"
              viewBox="0 0 20 20"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                clipRule="evenodd"
                d="M10 .333A9.911 9.911 0 0 0 6.866 19.65c.5.092.678-.215.678-.477 0-.237-.01-1.017-.014-1.845-2.757.6-3.338-1.169-3.338-1.169a2.627 2.627 0 0 0-1.1-1.451c-.9-.615.07-.6.07-.6a2.084 2.084 0 0 1 1.518 1.021 2.11 2.11 0 0 0 2.884.823c.044-.503.268-.973.63-1.325-2.2-.25-4.516-1.1-4.516-4.9A3.832 3.832 0 0 1 4.7 7.068a3.56 3.56 0 0 1 .095-2.623s.832-.266 2.726 1.016a9.409 9.409 0 0 1 4.962 0c1.89-1.282 2.717-1.016 2.717-1.016.366.83.402 1.768.1 2.623a3.827 3.827 0 0 1 1.02 2.659c0 3.807-2.319 4.644-4.525 4.889a2.366 2.366 0 0 1 .673 1.834c0 1.326-.012 2.394-.012 2.72 0 .263.18.572.681.475A9.911 9.911 0 0 0 10 .333Z"
                fillRule="evenodd"
              />
            </svg>
            <p
              className="py-1"
            >
              Mohammed Bennaceur
            </p>
          </a>
          <a
            className="flex flex-row items-center hover:text-accent dark:hover:text-white"
            href="https://github.com/Bolphunga"
            onClick={[Function]}
            onMouseEnter={[Function]}
            onTouchStart={[Function]}
          >
            <svg
              aria-hidden="true"
              className="w-4 h-4 mx-2"
              fill="currentColor"
              viewBox="0 0 20 20"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                clipRule="evenodd"
                d="M10 .333A9.911 9.911 0 0 0 6.866 19.65c.5.092.678-.215.678-.477 0-.237-.01-1.017-.014-1.845-2.757.6-3.338-1.169-3.338-1.169a2.627 2.627 0 0 0-1.1-1.451c-.9-.615.07-.6.07-.6a2.084 2.084 0 0 1 1.518 1.021 2.11 2.11 0 0 0 2.884.823c.044-.503.268-.973.63-1.325-2.2-.25-4.516-1.1-4.516-4.9A3.832 3.832 0 0 1 4.7 7.068a3.56 3.56 0 0 1 .095-2.623s.832-.266 2.726 1.016a9.409 9.409 0 0 1 4.962 0c1.89-1.282 2.717-1.016 2.717-1.016.366.83.402 1.768.1 2.623a3.827 3.827 0 0 1 1.02 2.659c0 3.807-2.319 4.644-4.525 4.889a2.366 2.366 0 0 1 .673 1.834c0 1.326-.012 2.394-.012 2.72 0 .263.18.572.681.475A9.911 9.911 0 0 0 10 .333Z"
                fillRule="evenodd"
              />
            </svg>
            <p
              className="py-1"
            >
              Ahmed Djebnoune
            </p>
          </a>
          <a
            className="flex flex-row items-center hover:text-accent dark:hover:text-white"
            href="https://github.com/samiraTbl"
            onClick={[Function]}
            onMouseEnter={[Function]}
            onTouchStart={[Function]}
          >
            <svg
              aria-hidden="true"
              className="w-4 h-4 mx-2"
              fill="currentColor"
              viewBox="0 0 20 20"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                clipRule="evenodd"
                d="M10 .333A9.911 9.911 0 0 0 6.866 19.65c.5.092.678-.215.678-.477 0-.237-.01-1.017-.014-1.845-2.757.6-3.338-1.169-3.338-1.169a2.627 2.627 0 0 0-1.1-1.451c-.9-.615.07-.6.07-.6a2.084 2.084 0 0 1 1.518 1.021 2.11 2.11 0 0 0 2.884.823c.044-.503.268-.973.63-1.325-2.2-.25-4.516-1.1-4.516-4.9A3.832 3.832 0 0 1 4.7 7.068a3.56 3.56 0 0 1 .095-2.623s.832-.266 2.726 1.016a9.409 9.409 0 0 1 4.962 0c1.89-1.282 2.717-1.016 2.717-1.016.366.83.402 1.768.1 2.623a3.827 3.827 0 0 1 1.02 2.659c0 3.807-2.319 4.644-4.525 4.889a2.366 2.366 0 0 1 .673 1.834c0 1.326-.012 2.394-.012 2.72 0 .263.18.572.681.475A9.911 9.911 0 0 0 10 .333Z"
                fillRule="evenodd"
              />
            </svg>
            <p
              className="py-1"
            >
              Samira TOUBAL SEGHIR
            </p>
          </a>
        </div>
      </div>
      <div
        className="flex justify-start flex-col uppercase tracking-wide "
      >
        <p
          className="font-semibold tracking-wider py-1"
        >
          Quick links
        </p>
        <a
          className="py-2 mt-3  no-underline hover:text-accent"
          href="#"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onTouchStart={[Function]}
        >
          <p>
            Home
          </p>
        </a>
        <a
          className="py-2  no-underline hover:text-accent"
          href="#"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onTouchStart={[Function]}
        >
          <p>
            Furniture
          </p>
        </a>
        <a
          className="py-2  no-underline hover:text-accent"
          href="#"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onTouchStart={[Function]}
        >
          <p>
            Clothes
          </p>
        </a>
        <a
          className="py-2  no-underline hover:text-accent"
          href="#"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onTouchStart={[Function]}
        >
          <p>
            Electronics
          </p>
        </a>
        <a
          className="py-2  no-underline hover:text-accent"
          href="#"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onTouchStart={[Function]}
        >
          <p>
            gaming
          </p>
        </a>
      </div>
      <div
        className=" flex justify-start flex-col uppercase"
      >
        <p
          className="font-semibold tracking-wide py-1"
        >
          ABOUT STUDENT STORE
        </p>
        <a
          className="py-2 no-underline hover:text-accent"
          href="#"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onTouchStart={[Function]}
        >
          <p>
            ABOUT US
          </p>
        </a>
        <a
          className="py-2 no-underline hover:text-accent"
          href="#"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onTouchStart={[Function]}
        >
          <p>
            HOW IT WORKS!
          </p>
        </a>
        <a
          className="py-2 no-underline hover:text-accent"
          href="#"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onTouchStart={[Function]}
        >
          <p>
            Terms of Use
          </p>
        </a>
        <a
          className="py-2  no-underline hover:text-accent"
          href="#"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onTouchStart={[Function]}
        >
          <p>
            Privacy Policy
          </p>
        </a>
      </div>
    </div>
  </div>
</footer>
`;
